q2_feature_classifier/_skl.py

```diff
@@ -21,7 +21,8 @@
             'non_negative': True}],
           ['classify',
            {'__type__': 'custom.LowMemoryMultinomialNB',
-            'alpha': 0.01}]]]]
+            'alpha': 0.01,
+            'fit_prior': False}]]]]
```

Comment:
The release notes might be:

- changed default taxon prior probabilities to uniform for taxonomy assignment, rather than estimating them from the reference database. This is in keeping with other standard approaches (including RDP Classifier). Users might notice small changes in classification confidence.

In future, is there a way that I can add to the release notes directly? Thanks!